src/jsonapi.js

```diff
@@ -145,16 +148,17 @@ export const updateResource = (resource) => {
   };
 };
 
-export const deleteResource = (resource) => {
+export const deleteResource = (resource, config = {}) => {
   return (dispatch, getState) => {
     dispatch(apiWillDelete(resource));
 
-    const { axiosConfig } = getState().api.endpoint;
+    const axiosConfig = getState().api.endpoint.axios;
```

Comment:
There seem to be a typo here, `endpoint.axios` should be `endpoint.axiosConfig`. If this isn't caught be any tests, we should make that happen 😄